installers/custom_ipxe/main_test.go

```diff
@@ -130,6 +130,25 @@ func TestIpxeScript(t *testing.T) {
 			echo userdata script
 			`,
 		},
+		{
+			"installer: ipxe data url",
+			"custom_ipxe",
+			&packet.InstallerData{Chain: "data:text/plain;charset=utf-8;base64,ZWNobyBteSBkYXRhIHVyaSBzY3JpcHQ="},
+			`#!ipxe
+
+
+			params
+			param body Device connected to DHCP system
+			param type provisioning.104.01
+			imgfetch ${tinkerbell}/phone-home##params
+			imgfree
+
+			set packet_facility test.facility
+			set packet_plan test.slug
+			
+			echo my data uri script
```

Comment:
The test fails, with this line being a chain-url line, referencing the unexpanded data uri.

Reply:
Can you paste the test failure here?

Reply:
I'm rerunning the test to capture the log output.

Reply:
```
--- FAIL: TestIpxeScript (0.00s)
    --- FAIL: TestIpxeScript/installer:_ipxe_data_url (0.00s)
        main_test.go:172:
            	Error Trace:	main_test.go:172
            	Error:      	Not equal:
            	            	expected: "#!ipxe\n\n\nparams\nparam body Device connected to DHCP system\nparam type provisioning.104.01\nimgfetch ${tinkerbell}/phone-home##params\nimgfree\n\nset packet_facility test.facility\nset packet_plan test.slug\n\necho my data uri script\n"
            	            	actual  : "#!ipxe\n\n\nparams\nparam body Device connected to DHCP system\nparam type provisioning.104.01\nimgfetch ${tinkerbell}/phone-home##params\nimgfree\n\nset packet_facility test.facility\nset packet_plan test.slug\nchain --autofree data:text/plain;charset=utf-8;base64,ZWNobyBteSBkYXRhIHVyaSBzY3JpcHQ=\n"

            	            	Diff:
            	            	--- Expected
            	            	+++ Actual
            	            	@@ -11,4 +11,3 @@
            	            	 set packet_plan test.slug
            	            	+chain --autofree data:text/plain;charset=utf-8;base64,ZWNobyBteSBkYXRhIHVyaSBzY3JpcHQ=

            	            	-echo my data uri script
            	            	-
            	Test:       	TestIpxeScript/installer:_ipxe_data_url
{"level":"error","ts":1637329705.027709,"caller":"custom_ipxe/main.go:54","msg":"validating ipxe config","service":"github.com/tinkerbell/boots","error":"ipxe config URL or Script must be defined"}
FAIL
```